addons/mrp_subcontracting/views/stock_picking_views.xml

```diff
@@ -12,7 +12,7 @@
             <xpath expr="//field[@name='move_ids_without_package']//tree//button[@name='action_show_details']" position="after">
                 <field name="show_subcontracting_details_visible" invisible="1"/>
                 <button name="action_show_subcontract_details" string="Register components for subcontracted product" type="object" icon="fa-sitemap"
-                    width="0" attrs="{'invisible': [('show_subcontracting_details_visible', '=', False)]}"/>
+                    width="0.1" attrs="{'invisible': [('show_subcontracting_details_visible', '=', False)]}"/>
```

Comment:
In the case of an icon button, a width in px would probably be more appropriate

Reply:
The problem with giving an absolute width is that a min-width will be set regardless of the existing data and take unnecessary space if the button is invisible. Also in this specific case it was to keep consistency with the two other buttons set to 0.1 (see `addons/stock/views/stock_picking_views.xml`)